Client/game_sa/CWaterManagerSA.cpp

```diff
@@ -839,12 +839,16 @@ bool CWaterManagerSA::TestLineAgainstWater(const CVector& vecStart, const CVecto
 
     // Check if we're outside of map area.
     CVector zeroPoint;
-    if (vecStart.IntersectsSegmentPlane(rayDir, CVector(0, 0, 1), CVector(0, 0, 0), &zeroPoint) && IsPointOutsideOfGameArea(zeroPoint))
+    float   waterHeight;
+    CVector pvecUnknown;
```

Comment:
Try passing nullptr for `vecUnknown`. It should work (judging from the source code..)

Reply:
I'll try that. Waiting for tests to pass.

Reply:
I mean the CI will pass, just test it locally, see if everything is as expected.